# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../slmodem/linux-2.6.18.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur slmodem-2.9.11-20051101-orig/drivers/amrmo_init.c slmodem-2.9.11-20051101/drivers/amrmo_init.c
--- slmodem-2.9.11-20051101-orig/drivers/amrmo_init.c	2005-10-10 00:14:13.000000000 +0300
+++ slmodem-2.9.11-20051101/drivers/amrmo_init.c	2006-09-21 10:42:50.000000000 +0300
@@ -53,7 +53,7 @@
 #include <linux/poll.h>
 #include <linux/fs.h>
 #include <asm/uaccess.h>
-#include <linux/devfs_fs_kernel.h>
+//#include <linux/devfs_fs_kernel.h>
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,5,0)
 #define OLD_KERNEL 1
diff -Nur slmodem-2.9.11-20051101-orig/drivers/kernel-ver.c slmodem-2.9.11-20051101/drivers/kernel-ver.c
--- slmodem-2.9.11-20051101-orig/drivers/kernel-ver.c	2003-09-23 20:38:40.000000000 +0300
+++ slmodem-2.9.11-20051101/drivers/kernel-ver.c	2006-09-21 00:16:44.000000000 +0300
@@ -4,7 +4,7 @@
  */
 
 #include <stdio.h>
-#include <linux/version.h>
+#include <linux/utsrelease.h>
 
 int main()
 {
diff -Nur slmodem-2.9.11-20051101-orig/drivers/st7554.c slmodem-2.9.11-20051101/drivers/st7554.c
--- slmodem-2.9.11-20051101-orig/drivers/st7554.c	2005-11-01 17:31:45.000000000 +0200
+++ slmodem-2.9.11-20051101/drivers/st7554.c	2006-09-21 11:20:56.000000000 +0300
@@ -55,7 +55,7 @@
 #include <linux/poll.h>
 #include <linux/usb.h>
 #include <linux/device.h>
-#include <linux/devfs_fs_kernel.h>
+//#include <linux/devfs_fs_kernel.h>
 
 #include <modem_defs.h>
 
